package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@datastax/astra-db-ts",
```

Comment:
The issue will persist unless the name here is changed too or the `update-version-file` script is updated to not read this field